src/script/descriptor.cpp

```diff
@@ -886,6 +936,13 @@ std::unique_ptr<PubkeyProvider> ParsePubkeyInner(uint32_t key_exp_index, const S
                     error = "Uncompressed keys are not allowed";
                     return nullptr;
                 }
+            } else if (data.size() == 32 && ctx == ParseScriptContext::P2TR) {
+                unsigned char fullkey[33] = {0x02};
+                std::copy(data.begin(), data.end(), fullkey + 1);
+                pubkey.Set(std::begin(fullkey), std::end(fullkey));
+                if (pubkey.IsFullyValid()) {
+                    return MakeUnique<ConstPubkeyProvider>(key_exp_index, pubkey);
```

Comment:
nit: in d5a0dce8fe40153b07ec6e882e8b1568b05e3905: we can use `std::make_unique` in new code.

Reply:
Done.